src/app/shared/layout/header.component.html

```diff
@@ -22,8 +22,8 @@
   </div>
 
   <span style="flex: 1 1 auto"></span>
-  <button *ngIf="auth.isAuthenticated() && isOpenUrlButtonShown()" mat-button matTooltip="View current page in web-browser" (click)="viewBrowser()">
-    View in Browser
+  <button *ngIf="auth.isAuthenticated() && isOpenUrlButtonShown()" mat-button matTooltip="View current page on Github" (click)="viewBrowser()">
+    View on Github
```

Comment:
```suggestion
    View on GitHub
```

Sorry - just saw this typo - let's get the display right if we're updating this